test_genomicsdbjar.sh

```diff
@@ -43,8 +46,10 @@ fi
 echo "Done"
 echo
 
+GENOMICSDB_TAG=${GENOMICSDB_TAG:-v$GENOMICSDB_VERSION}
```

Comment:
minor: I guess we only need one of `GENOMICSDB_TAG` or `GENOMICSDB_VERSION` not both here...?

Reply:
I would like to keep `GENOMICSDB_TAG` separate from `GENOMICSDB_VERSION` to check for backward compatibility. That way, we will be required to bump the major/minor version when the backward compatibility is broken. Can augment this https://github.com/GenomicsDB/GenomicsDB/pull/282 with a backward compatibility check? Does that sound OK?